### PR TITLE
doc: Add introduction for using Xenomai VM with Industry scenario

### PR DESCRIPTION
**Pull request by totoroow**
Signed-off-by: Tw <wei.tan@intel.com>

### CODE REVIEW

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by wenlingz** [comment]
@totoroow  does this PR for release_1.6 ? we checked that this PR includes source code, generally speaking, doc and source code need pull request separately, and code need pass weekly test.
@NanlinXie  what's your opinion

**Comment by NanlinXie** [comment]
Agree with split PR for code and doc. @totoroow , suggest the xenomai VM as separate doc below menu “Advanced Guides” -> User VM Tutorials, and you may have some link back to GSG for the repetition.

**Comment by totoroow** [comment]
OK, I will split this PR into two parts for script code and document respectively.

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

File: doc/tutorials/using_xenomai_as_uos.rst [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+      drwxr-xr-x  20 tw tw    12288 Feb 25 16:17 kernel
+      drwxr-xr-x  13 tw tw    20480 Feb 21 11:01 lib
+      drwxr-xr-x   5 tw tw     4096 Feb 20 10:17 LICENSES
+      -rw-r--r--   1 tw tw 17572590 Feb 25 16:17 linux-4.19.59-xenomai-3.1-tw+-x86.tar.gz
+      ...
+
+Launch RTVM
+***********
+
+#. Prepare a dedicated disk (NVMe or SATA) for RTVM, just as you did :ref:`for Preempt-RT <install_rtvm>`, here we suppose it's ``/dev/sda1``.
```
Comment (RonnieYang) [inline doc/tutorials/using_xenomai_as_uos.rst]: Suppose we only support Xenomai on SATA disk, NVMe is currently not supported yet?

File: doc/tutorials/using_xenomai_as_uos.rst [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+#. Then you can launch RTVM via our script. We need to tell it where the storage device and kernel tarball are::
+
+	# /usr/share/acrn/samples/nuc/launch_xenomai.sh -b /dev/sda1 -k /path/to/linux-4.19.59-xenomai-3.1-tw+-x86.tar.gz
```
Comment (RonnieYang) [inline doc/tutorials/using_xenomai_as_uos.rst]: In our internal tests, SATA disk will have 3 partitions and rootfs will be /dev/sda3

**Pull request closed without merging**